### PR TITLE
7080150: [TEST_BUG][macosx] No mac os x EmbeddedFrame support in regression regtesthelpers

### PR DESCRIPTION
**Pull request by alisenchung**
added EmbeddedFrame support for macosx in regtesthelpers

---------
### Progress
- [ ] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.java.net/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-7080150](https://bugs.openjdk.java.net/browse/JDK-7080150): [TEST_BUG][macosx] No mac os x EmbeddedFrame support in regression regtesthelpers

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/8348/head:pull/8348` \
`$ git checkout pull/8348`

Update a local copy of the PR: \
`$ git checkout pull/8348` \
`$ git pull https://git.openjdk.java.net/jdk pull/8348/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 8348`

View PR using the GUI difftool: \
`$ git pr show -t 8348`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/8348.diff">https://git.openjdk.java.net/jdk/pull/8348.diff</a>

</details>

### CODE REVIEW

File: test/jdk/java/awt/regtesthelpers/UtilInternal.java [hunk 1]
```diff
@@ -73,6 +73,12 @@ public static Frame createEmbeddedFrame(final Frame embedder)
             Class clazz = Class.forName("sun.awt.X11.XEmbeddedFrame");
             Constructor constructor = clazz.getConstructor (new Class [] {Long.TYPE, Boolean.TYPE});
             return (Frame) constructor.newInstance (new Object[] {window, true});
+        } else if ("sun.lwawt.macosx.LWCToolkit".equals(tk.getClass().getName())) {
+            java.awt.Helper.addExports("sun.lwawt", UtilInternal.class.getModule());
+            Class window_peer_class = Class.forName("sun.lwawt.LWComponentPeer");
+            Method get_target = window_peer_class.getMethod("getTarget", new Class[0]);
+            System.out.println("get_target = " + get_target);
+            return (Frame) get_target.invoke(frame_peer, new Object[0]);
```
Comment (prsadhuk) [inline test/jdk/java/awt/regtesthelpers/UtilInternal.java:81]: Shouldn't we invoke sun/lwawt/macosx/CEmbeddedFrame same as we do for windows/linux via WEmbeddedFrame and XEmbeddedFrame?

Comment (alisenchung) [reply test/jdk/java/awt/regtesthelpers/UtilInternal.java:81]: CEmbeddedFrame doesn't have a way to pass in a handle like in WEmbeddedFrame and XEmbeddedFrame, so I can't pass in the peer's target. If I try to use CEmbeddedFrame the test window will pop up but the EmbeddedFrame won't show up and the test fails

Comment (mrserb) [reply test/jdk/java/awt/regtesthelpers/UtilInternal.java:81]: Are you sure that CEmbeddedFrame is used in the embedding functionality? We have another one on macOS CViewEmbeddedFrame. One of them was used for applets another one was used by the JAWT and SWT. Please double-check that the correct one is used for this test, and another one probably can be removed.

Comment (prsadhuk) [reply test/jdk/java/awt/regtesthelpers/UtilInternal.java:81]: It seems LWCToolkit creates a CViewPlatformEmbeddedFrame for VIEW_EMBEDDED_FRAME type so probably you can call CViewPlatformEmbeddedFrame.getNSViewPtr to get the long value and then call CViewEmbeddedFrame with the nsviewPtr value

**Pull request closed without merging**